src/anm/utils.js

```diff
@@ -12,7 +12,7 @@ is.defined = function(v) {
 is.finite = global.isFinite;
 is.nan = global.isNaN;
 is.arr = Array.isArray;
-is.int = function(n) {
+is.integer = function(n) {
```

Comment:
I renamed `is.int` because closure compiler complained about usage of reserved word `int`, other methods are fine for now. I'll make a note to rename them later, consistency is nice, yeah.